test/helpers/kubectl.go

```diff
@@ -2219,9 +2228,9 @@ func (kub *Kubectl) overwriteHelmOptions(options map[string]string) error {
 		}
 
 		opts := map[string]string{
-			"global.kubeProxyReplacement": "strict",
-			"global.k8sServiceHost":       nodeIP,
-			"global.k8sServicePort":       "6443",
+			"kubeProxyReplacement": "strict",
+			"k8sServiceHost":       nodeIP,
+			"k8sServicePort":       "6443",
```

Comment:
One more just below, there is `config.bpfMasquerade` which needs converting over to the new approach.